src/components/Header/Header.js

```diff
@@ -1,8 +1,9 @@
 import React from 'react';
 
-const Header = () => {
+const Header = ({selectedId, selectMovie}) => {
   return (
     <header>
+      {selectedId !== 0 && <button onClick={() => selectMovie(0)}>HOME</button>}
```

Comment:
So one note I wanna leave on some specific code is that I think we can move some of this logic to `<App />`. We can do that later though.